credentials.go

```diff
@@ -135,5 +130,12 @@ func (c *Credentials) String() string {
 			Value: "true",
 		})
 	}
+
+	pp = append(pp, param.Param{
+		Key:   "response",
+		Value: c.Response,
+		Quote: true,
+	})
+
```

Comment:
Remove the surrounding blank lines and add a comment (with a link to the issue) describing why this needs to be a the end. Otherwise someone might forget and move it back.